go/mysql/mysql56_gtid_set.go

```diff
@@ -579,6 +580,36 @@ func (set Mysql56GTIDSet) Difference(other Mysql56GTIDSet) Mysql56GTIDSet {
 	return differenceSet
 }
 
+func (set Mysql56GTIDSet) NumTransactions() (int64, error) {
+	txs := int64(0)
+	for _, intervals := range set {
+		for _, interval := range intervals {
+			txs += interval.end - interval.start + 1
+		}
+	}
+	return txs, nil
+}
+
+func (set Mysql56GTIDSet) Distance(other GTIDSet) (int64, error) {
+	mydbOther, ok := other.(Mysql56GTIDSet)
```

Comment:
Would it be better to perform a subtract, and then count the num transactions? It will be more accurate.

Reply:
Changed to count the transactions reported by the Difference() function.